### ISSUE TITLE
skypeweb: left groupchats notify on new connection

### ISSUE THREAD

**Issue by psukys** [EionRobb/skype4pidgin#408]
I am using Pidgin 2.10.9 and the plugin version is 1.1

After leaving a groupchat (via /leave command), whenever I run Pidgin (cold run, when having internet connection) or connect to internet, when Pidgin is on, I get a notification (in Pidgin's tabs) from that room and after clicking on it, it just says that I've left the chat. I've checked with people that did not leave the chat and tried "remote testing" by asking to write a message, but it did not trigger the notification.

TL;DR Steps to reproduce:
1. Join groupchat in skype
2. Leave groupchat
3. Exit pidgin (as in shutting down the program) or turn internet connection off and on
4. Verify recently left groupchat still notifies in Pidgin
5. Repeat steps 3 and 4 to reproduce

**Comment by psukys** [comment]
Can anyone verify this?

**Comment by EionRobb** [comment]
Unfortunately there's a timing issue with event history.  The server wants the last event time in milliseconds, but Pidgin only provides the time in seconds.  This can mean that sometimes old events will display in a conversation, such as "xyz has left the conversation" events.  I might be able to tweak it to specifically remove the leave notifications, but I'd rather try and come up with a solution for millisecond-accuracy in history, as that would help with other bugs in the plugin (like marking the last message of a conversation as unread on the server)
